app/javascript/packs/public.js

```diff
@@ -63,7 +63,7 @@ function main() {
       content.textContent = timeAgoString({
         formatMessage: ({ id, defaultMessage }, values) => (new IntlMessageFormat(messages[id] || defaultMessage, locale)).format(values),
         formatDate: (date, options) => (new Intl.DateTimeFormat(locale, options)).format(date),
-      }, datetime, now, datetime.getFullYear());
+      }, datetime, now, now.getFullYear());
```

Comment:
This is an unrelated fix. Public pages were displaying "Nov 28" even for dates 2 years back, unlike the web UI, which correctly displayed "Nov 28, 2016", because of this mistake